locations/spiders/rexel_gb.py

```diff
@@ -0,0 +1,12 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class RexelGBSpider(RexelSpider):
+    name = "rexel_gb"
+    item_attributes = {"brand": "Rexel", "brand_wikidata": "Q"}
```

Comment:
```suggestion
    item_attributes = {"brand": "Rexel", "brand_wikidata": "Q962489"}
```

Reply:
I'd seen the QID but wasn't sure if it was French only.

I just went to do this, but you've beat me to it! :
https://github.com/osmlab/name-suggestion-index/commit/4d07c92457d0c7f2c1d1b6048228a8d6f37a90d8